services/app-api/src/resolvers/contract/contractResolver.ts

```diff
@@ -32,7 +33,22 @@ export function contractResolver(store: Store): Resolvers['Contract'] {
         },
 
         draftRevision(parent) {
-            return parent.draftRevision || {}
+            const programsForContractState = statePrograms.states
+                .find((state) => state.code === parent.stateCode)
+                ?.programs.filter((program) => program !== undefined)
```

Comment:
How do we feel about this? Making a backend change to fix design bug with headings (which are either package or rate names when we in the form). 

I was debating also just putting this directly on the `ContractFormData` but thats a bigger code change that touches more files. Thought I would throw this  up and get feedback

Reply:
I think it's to approach it like this but I do think having it on `ContractFormData` would be nice so that it's similar to the rate name